ngrok-java-native/src/lib.rs

```diff
@@ -580,7 +588,11 @@ impl<'local> NativeSessionRsImpl<'local> {
         bldr.proxy_proto(ProxyProto::from(jeb.get_proxy_proto_version(self.env)));
 
         if let Some(policy) = jeb.get_policy(self.env).of_string(self.env) {
```

Comment:
shouldn't we just drop this, it is still just the `traffic_policy` right?

Reply:
uhhhh.....
yeah i think we could. I was trying to cover for the case where they set the member variable directly, but I forgot that Java has strong encapsulation, so these are only available through getters and setters. Which means we shooooould be able to remove it? 
But if it's truly doing nothing, we can also just leave it and rip it out when we actually do the real deprecation of `policy` at some point in...the next few months?

@nikolay-ngrok if you're ok leaving it until then then we can. If you'd rather it be removed now, lmk.

Reply:
🤷 happy to go either way 👍